frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { useEffect, useState } from 'react'
+import Col from 'react-bootstrap/Col'
+import Form from 'react-bootstrap/Form'
+import Card from 'react-bootstrap/Card'
+import Button from 'react-bootstrap/Button'
+import { useParams } from 'react-router-dom'
+import style from './HospitalInventoryEdit.module.css'
+import { HospitalInventory, HospitalAddress, fetchHospitalInventory, fetchHospitalAddress } from '../apis/infomed'
+
+
+interface HospitalInventoryParams {
+  hospitalId: string
+}
+
+export const HospitalInventoryEdit = ({hospitalId}: HospitalInventoryParams) => {
+
+  const [inventoryList, setInventoryList] = useState<HospitalInventory[]>([]);
+  const [hospitalAddress, setHospitalAddress] = useState<HospitalAddress[]>([]);
+
+  useEffect(() => {
+    (async () => {
+      const list = await fetchHospitalInventory(hospitalId);
+      setInventoryList(list);
+      
+      const address = await fetchHospitalAddress(hospitalId);
+      setHospitalAddress(address);
+    })();
+  }, [hospitalId]);
+
+  const handleInventoryChange  = (e: React.ChangeEvent<HTMLInputElement>): void => {
+    const updateList = [...inventoryList];
```

Comment:
Nitpick: `updateList` is a verb, which suggests "this is a function". `updatedList` is better, because is't a noun, which suggests "this is an object". :)